An error occurs during installation of event-3.0.7

### Version of install-php-extensions

install-php-extensions v.1.5.19

### Error description

Just as the title describes, it reports an error.

```
...
...
Starting to download event-3.0.7.tgz (199,516 bytes)
.........................................done: 199,516 bytes
76 source files, building
running: phpize
Configuring for:
PHP Api Version:         20170718
Zend Module Api No:      20170718
Zend Extension Api No:   320170718
Enable internal debugging in Event [no] : Enable sockets support in Event [yes] : libevent installation prefix [/usr] : Include libevent's pthreads library and enable thread safety support in Event [no] : Include libevent protocol-specific functionality support including HTTP, DNS, and RPC [yes] : Include libevent OpenSSL support [yes] : PHP Namespace for all Event classes [no] : openssl installation prefix [no] : building in /tmp/pear/temp/pear-build-defaultuserhwGHka/event-3.0.7
running: /tmp/pear/temp/event/configure --with-php-config=/usr/local/bin/php-config --enable-event-debug=no --enable-event-sockets=no --with-event-libevent-dir=/usr --with-event-pthreads --with-event-extra --with-event-openssl --with-event-ns=no --with-openssl-dir=no
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for a sed that does not truncate output... /bin/sed
checking for cc... cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether cc accepts -g... yes
checking for cc option to accept ISO C89... none needed
checking how to run the C preprocessor... cc -E
checking for icc... no
checking for suncc... no
checking whether cc understands -c and -o together... yes
checking for system library directory... lib
checking if compiler supports -R... no
checking if compiler supports -Wl,-rpath,... yes
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking target system type... x86_64-unknown-linux-gnu
checking for PHP prefix... /usr/local
checking for PHP includes... -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib
checking for PHP extension directory... /usr/local/lib/php/extensions/no-debug-non-zts-20170718
checking for PHP installed headers prefix... /usr/local/include/php
checking if debug is enabled... no
checking if zts is enabled... no
checking for re2c... re2c
checking for re2c version... 1.1.1 (ok)
checking for gawk... no
checking for nawk... nawk
checking if nawk is broken... no
checking for Event core support... yes, shared
checking for Event thread safety support... yes
checking for Event extra functionality support... yes
checking for OpenSSL support in Event... yes
checking for custom PHP namespace in Event... no
checking for OpenSSL installation prefix... no
checking whether Event debugging support enabled... no
checking whether to enable sockets support in Event... no
checking PHP version... PHP 7.x
checking for include/event2/event.h... found in /usr
checking for libevent version... ok
checking for event_free in -levent_core... yes
checking for evthread_use_pthreads in -levent_pthreads... yes
checking for evdns_base_free in -levent_extra... yes
checking for pkg-config... /usr/bin/pkg-config
configure: error: Cannot find OpenSSL's <evp.h>
ERROR: `/tmp/pear/temp/event/configure --with-php-config=/usr/local/bin/php-config --enable-event-debug=no --enable-event-sockets=no --with-event-libevent-dir=/usr --with-event-pthreads --with-event-extra --with-event-openssl --with-event-ns=no --with-openssl-dir=no' failed
```

### Docker image

php:7.2-cli

### Minimal Dockerfile

```Dockerfile
FROM php:7.2-cli

ENV TZ=Asia/Shanghai

ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/

RUN set -eux; \
    chmod +x /usr/local/bin/install-php-extensions; \
    install-php-extensions \
    event-3.0.7
```

--- a/install-php-extensions
+++ b/install-php-extensions
@@ -2481,7 +2481,7 @@ installRemoteModule() {
 					addConfigureOption with-event-ns no
 				fi
 				# openssl installation prefix
-				addConfigureOption with-openssl-dir no
+				addConfigureOption with-openssl-dir yes
 			elif test $(compareVersions "$installRemoteModule_version" 1.7.6) -ge 0; then
 				# Enable internal debugging in Event
 				addConfigureOption enable-event-debug no

